assets/js/lib/analytics/pageview.js

```diff
@@ -7,11 +7,19 @@ import { getUserProfile } from '@state/selectors/user';
 export default function PostHogPageView() {
   const location = useLocation();
   const { analytics_enabled } = useSelector(getUserProfile);
+  const webversion = document
```

Comment:
I think we can use the `getFromConfig` function outside of the component itself, as we do in many other cases